raiden/api/rest.py

```diff
@@ -497,13 +519,20 @@ def open(
 
     def connect(
             self,
-            registry_address,
-            token_address,
-            funds,
-            initial_channel_target=None,
-            joinable_funds_target=None,
+            registry_address: typing.PaymentNetworkID,
+            token_address: typing.TokenAddress,
+            funds: typing.TokenAmount,
+            initial_channel_target: int = None,
+            joinable_funds_target: float = None,
     ):
-
+        log.debug(
+            '[REST API] Connecting to token network',
```

Comment:
Can we configure the logger to add the module_name that's logging instead of appending `[REST API]` into each log line?

Reply:
That should already be there, I thought it might be nice. But on second thought doesn't make much sense.